src/TuxOwneeClass.js

```diff
@@ -0,0 +1,23 @@
+var React = require('react');
+var getOwnerPropsMixin = require('./TuxGetOwnerPropsMixin');
+var assign = require('object-assign');
+
+//createOwneeClass FUNCTION creates an ownee Tux Class which is a type of React class not designed to manage any of its own state other than, in some cases, necessary state to implement two-way data-binding for input validation
+//@param owneeClassProps OBJECT: properties that the new owneeClass will possess, should be a standard React.createClass object with properties such as componentWillMount, componentDidMount, etc.  Please see http://facebook.github.io/react/docs/component-specs.html for a full list of React props
```

Comment:
Recommend mentioning the requirement of the render method on the React.createClass object.